seed/challenges/object-oriented-and-functional-programming.json

```diff
@@ -81,105 +88,164 @@
         "",
         "",
         "",
-        "};",
-        "",
+        "};"
+      ],
+      "tail":[
         "var myMotorBike = new MotorBike();",
-        "",
-        "// Only change code above this line.",
-        "",
         "(function() {return JSON.stringify(myMotorBike);})();"
       ],
+      "solutions":[
+        "var Car = function() {\n  this.wheels = 4;\n  this.engines = 1;\n  this.seats = 1;\n};\n\nvar myCar = new Car();\n\nvar MotorBike = function() {\n  this.engines = 1;\n  this.seats = 1;\n  this.wheels = 4;\n};\n\nvar myMotorBike = new MotorBike();"
+      ],
       "challengeType":1,
       "type": "waypoint"
     },
     {
-      "id":"cf1111c1c15feddfaeb3bdef",
-      "title":"Make Object Properties Private",
+      "id":"cf1111c1c15feddfaeb4bdef",
+      "title":"Make Instances of Objects with a Constructor Function",
```

Comment:
Is this a new challenge or a re-write? If it is, please you MongoDB to create a new ObjectID instead of incrementing the number.